### PR TITLE
Docs: Clear guide for package manager installation

### PR DESCRIPTION
**Pull request by shivam-sharma7**

### CODE REVIEW

**Comment by shivam-sharma7** [comment]
@melloware

**Comment by ia3andy** [comment]
AFAIK the node version is also required

**Comment by melloware** [comment]
You mean the NPM version?

File: docs/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -10,7 +10,13 @@ Live code the backend and frontend together with close to no configuration. When
 
 * Create or use an existing Quarkus application
 * Add the Quinoa extension
-* Install NodeJS (https://nodejs.org/) or make sure Quinoa is link:config-reference.adoc#quarkus-quinoa_quarkus.quinoa.package-manager-install[configured] to install it.
+* Install NodeJS (https://nodejs.org/) or make sure Quinoa is https://docs.quarkiverse.io/quarkus-quinoa/dev/config-reference.html#quarkus-quinoa_quarkus.quinoa.package-manager-install[configured] to install it, see the following example.
+
+----
+quarkus.quinoa.package-manager-install=true
```
Comment (melloware) [inline docs/modules/ROOT/pages/index.adoc:16]: You don't need the `install=true` step anymore in 2.2.1 we detect any change in package.json and we automatically run on the first instance.

**Review by melloware — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (shivam-sharma7) [reply docs/modules/ROOT/pages/index.adoc]: Tested it on Horreum and worked fine. Thanks :)

**Review by melloware — APPROVED** [review]

**Comment by shivam-sharma7** [comment]
@melloware @ia3andy a fresh git clone not working with above config, me and @jesperpedersen has tested

**Comment by melloware** [comment]
You mean you checkout and run mvn clean package and it fails?

**Comment by shivam-sharma7** [comment]
> You mean you checkout and run mvn clean package and it fails?

Yes, on Horreum

**Comment by melloware** [comment]
Yearn failing for me.

````
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.444 s <<< FAILURE! -- in io.quarkiverse.quinoa.test.QuinoaPackageManagerSetYarnConfigTest
[ERROR] io.quarkiverse.quinoa.test.QuinoaPackageManagerSetYarnConfigTest -- Time elapsed: 1.444 s <<< ERROR!
java.lang.RuntimeException:
java.lang.RuntimeException: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
        [error]: Build step io.quarkiverse.quinoa.deployment.QuinoaProcessor#install threw an exception: java.lang.RuntimeException: Error in Quinoa while running package manager install command: yarn.cmd install
```

**Pull request merged**